src/modules/AuthModule/components/ForgetPass/ForgetPass.tsx

```diff
@@ -1,8 +1,163 @@
-import React from 'react'
+import {
+  Alert,
+  InputLabel,
+  OutlinedInput,
+} from "@mui/material";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import CssBaseline from "@mui/material/CssBaseline";
+import Grid from "@mui/material/Grid";
+import Link from "@mui/material/Link";
+import Paper from "@mui/material/Paper";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import Typography from "@mui/material/Typography";
+import { emailValidation } from "./../../../../validations/validations";
+import { SubmitHandler, useForm } from "react-hook-form";
+import { FormValuesForgetPass } from "../../../../interfaces/Auth";
+import { axiosInstance } from "../../../../axiosConfig/axiosInstance";
+import { useState } from "react";
+import { toast } from "react-toastify";
+import { useNavigate } from "react-router-dom";
+
+function Copyright(props: any) {
+  return (
+    <Typography
+      variant="body2"
+      color="text.secondary"
+      align="center"
+      {...props}
+    >
+      {"Copyright © "}
+      <Link href="https://mui.com/">Your Website</Link>{" "}
+      {new Date().getFullYear()}
+      {"."}
+    </Typography>
+  );
+}
+
+// TODO remove, this demo shouldn't need to reset the theme.
+const defaultTheme = createTheme();
 
 const ForgetPass = () => {
+
+  const [loadingBtn, setLoadingBtn] = useState(false);
+  const navigate = useNavigate();
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<FormValuesForgetPass>();
+
+  const backgroundStyle = {
+    backgroundSize: "cover",
+    backgroundPosition: "center",
+    backgroundRepeat: "no-repeat",
+    display: { xs: "none", sm: "flex" },
+    flexDirection: "column",
+    justifyContent: "flex-end",
+    alignItems: "center",
+  };
+
+  const onSubmit: SubmitHandler<FormValuesForgetPass> = async (data) => {
+    // setLoadingBtn(true);
+    try {
+      const response = await axiosInstance.post("admin/users/forgot-password", data);
+      console.log(response);
+      toast.success(response.data.message ||"check your email");
+      navigate("/reset-pass");
+    } catch (error : any) {
+      console.log(error);
+      toast.error(error.response.data.message);
+    } 
+    console.log(data);
+  };
+
   return (
-    <div>ForgetPass</div>
+    <>
+    <ThemeProvider theme={defaultTheme}>
```

Comment:
The theme provided should be at the top of components tree, we can move it to the App.